Affine Cipher.py

```diff
@@ -1,38 +1,53 @@
 # v0.9
+# infinite loop
+while 0 == 0:
+    result = ""
 
-result = ""
 
-# This function is a modified modular multiplicative inverse
-def modInverse(y):
-    for x in range(0, 26):
-        if (9 * x) % 26 == ((y - 7) % 26):
-            return x
-    return x
+    # This function is a modified modular multiplicative inverse
+    def modInverse(y):
+        for x in range(0, 26):
+            if (9 * x) % 26 == ((y - 7) % 26):
+                return x
+        return x
 
 
-select = input("Type E for encryption and D for decryption")
-code_input = input("Enter the Word")
+    # input
+    select = input("Enter E for encryption or D for decryption: ")
+    if select == "E":
+        code_input = input("Enter word to be encoded: ")
+    elif select == "D":
+        code_input = input("Enter word to be decoded: ")
+    else:
+        code_input = "error"
```

Comment:
just do 
`if select == "E" or select == "D"`

Reply:
No. The whole point of me changing that was so it says "encoded" or "decoded", depending on which one the user selects.

Reply:
Oh I see. I didn't see that before.